hkm/management/commands/clean_unused_data.py

```diff
@@ -22,8 +22,8 @@ def handle(self, *args, **kwargs):
         self.stdout.write("Old data cleaning started! Removing data older than " + str(days) + " days.")
         counted_date = timezone.now() - timedelta(days=days)
 
-        # Find and delete unused users
-        users = User.objects.filter(last_login__lte=counted_date).delete()
+        # Find and delete unused users. Do not delete superusers or staff users.
+        users = User.objects.filter(last_login__lte=counted_date, is_superuser=False, is_staff=False).delete()
```

Comment:
It's possible that there are users who are marked with the ´museum´ role and are not superusers/staff. This should be verified before this change is finished.

Reply:
Finally found the correct way to reference `User profile`, which holds `is_museum` and `is_admin` fields. They are now ignored as well.